README.md

```diff
@@ -24,11 +24,14 @@ The easiest way to contribute to OpenEar is creating some new exercises on the e
 To contribute follow the following steps.
 
 1. Fork the repository
-2. open the repository root
-3. Install ionic globally by running `npm i ionic -g`
-4. Run `yarn` or `npm i` (you may need to install node if you don't have it installed already)
-5. Run `yarn start`
-6. App should be served in the browser!
+2. Open the repository root
+3. Download and install node at https://nodejs.org/en/download/
+4. Open nodejs directory
```

Comment:
Why is this needed?

Reply:
For me it didn't worked at first but when I changed directory to nodejs it did.

Reply:
You mean "npm" was not defined as an [Environment Variable](https://en.wikipedia.org/wiki/Environment_variable) ?
That's something that should have worked after the installation, and is definitely not a required part of node installation was done properly. 

Anyway we can remove this step from the readme.

Reply:
Fixed